scss/widgets/generic/checkBox/_index.scss

```diff
@@ -69,17 +69,6 @@ $generic-checkbox-disable-opacity: 0.4;
   }
 }
 
-.dx-checkbox-text {
-  margin-left: -$generic-checkbox-size;
```

Comment:
why did u remove text styles?

Reply:
i do not see any sence of this margin, when checkBox size is not static anymore. made the same behaviour now by using flex